lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1091,6 +1091,11 @@ LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
       /*out=*/remPipe.out_remainder());
 }
 
+// https://github.com/llvm/circt/issues/2660
+static inline unsigned handleZeroWidth(int64_t dim) {
```

Comment:
+1 to what @mikeurbach said. Don't use inline in the .cpp file. It can result in naming conflicts that may have otherwise been caught by the linker.